rll-examples/install-packages.sh

```diff
@@ -0,0 +1,114 @@
+#! /bin/bash
+# ---
+# RightScript Name: SYS packages install
+# Description: |
+#   Installs packages required by RightScripts.
+#   To handle naming variations, prefix package names with "yum:" or "apt:".
+# Inputs:
+#   PACKAGES_INSTALL:
```

Comment:
I like the idea of removing the input entirely since if you are running this RightScript, you are doing it for a reason.  I also mentioned below, if `$RS_PACKAGES` is empty, can we just exit out without an error?